src/test/java/com/iexec/worker/compute/ComputeManagerServiceTests.java

```diff
@@ -376,73 +381,28 @@ public void shouldRunTeePostComputeWithFailureResponse() {
     }
 
     // computeImagePullTimeout
-    @Test
-    void computeImagePullTimeoutForXSCategory() {
-        final TaskDescription taskDescription = TaskDescription
-                .builder()
-                .maxExecutionTime(3000)
-                .build();
-
-        when(dockerRegistryConfiguration.getMinPullTimeout()).thenReturn(60L);
-        when(dockerRegistryConfiguration.getMaxPullTimeout()).thenReturn(600L);
-
-        Assertions.assertThat(computeManagerService.computeImagePullTimeout(taskDescription))
-                .isEqualTo(60);
-    }
-
-    @Test
-    void computeImagePullTimeoutForSCategory() {
-        final TaskDescription taskDescription = TaskDescription
-                .builder()
-                .maxExecutionTime(12000)
-                .build();
-
-        when(dockerRegistryConfiguration.getMinPullTimeout()).thenReturn(60L);
-        when(dockerRegistryConfiguration.getMaxPullTimeout()).thenReturn(600L);
-
-        Assertions.assertThat(computeManagerService.computeImagePullTimeout(taskDescription))
-                .isEqualTo(240);
+    static Stream<Arguments> computeImagePullTimeoutValues() {
+        return Stream.of(
+                Arguments.of(3000, 419),        // XS category
```

Comment:
You might add 2 fictive categories XXS and XXL to see if min and max works, but otherwise it's great!

Reply:
Sure, see https://github.com/iExecBlockchainComputing/iexec-worker/pull/442/commits/2c95f49a424dc7fdb2164edf6e86b7244898257b.

Reply:
I don't understand the need for fictive categories. Why do we need cases that do not exist ? Is it not over-specified ?

Reply:
More precisely, I understand why they are useful for tests and I agree with parameterized tests.
I would avoid to mention them in javadoc as those categories do not exist.
Otherwise, it could be possible to use min and max timeout values to have XS category below min timeout and XL category beyond max timeout.

Reply:
Changing min and max values is more representative of what a user will be facing.
I would prefer to have tests with several values for min and max.
What should happen if by any chance we have a situation where `image.pull-timeout.max < image.pull-timeout.min` ?

Reply:
OK, done in https://github.com/iExecBlockchainComputing/iexec-worker/pull/442/commits/ebbe37f9bfa780eaa1fc82b5bc36a8f941a758f3.

Reply:
Thanks for the last commit.
Shouldn't the new condition in javadoc be inverted ?
`less` seems to be 'wrong' here

If {@link DockerRegistryConfiguration#getMinPullTimeout()} is less than {@link DockerRegistryConfiguration#getMaxPullTimeout()},

Reply:
Good catch, fixed. Thanks!